vpr/src/route/rr_graph.h

```diff
@@ -23,7 +23,7 @@ enum {
 void create_rr_graph(const t_graph_type graph_type,
                      const std::vector<t_physical_tile_type>& block_types,
                      const DeviceGrid& grid,
-                     t_chan_width nodes_per_chan,
+                     const t_chan_width& nodes_per_chan,
```

Comment:
Could also pass det_routing_arch by reference.  Or const ref if we don't update this in rr graph building.